python/ray/data/_internal/util.py

```diff
@@ -513,3 +513,20 @@ def unify_block_metadata_schema(
         # return the first schema.
         return schemas_to_unify[0]
     return None
+
+
+def get_attribute_from_class_name(class_name: str) -> Any:
```

Comment:
Checked online, it looks like it's the recommended way to do it - https://stackoverflow.com/questions/452969/does-python-have-an-equivalent-to-java-class-forname .